SkietbaanBE/SkietbaanBE/Program.cs

```diff
@@ -14,12 +14,13 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            Console.Write(Environment.GetEnvironmentVariable("TEMP"));
```

Comment:
Is TEMP for testing purposes?

Reply:
Also, shouldn't the consoles be taken out?

Reply:
Ja I wanted to show someone how we could use the environment variable to store the connection string. Then i forgot to remove it